modules/accessing-notebook-servers-owned-by-other-users.adoc

```diff
@@ -0,0 +1,31 @@
+:_module-type: PROCEDURE
+
+[id='accessing-notebook-servers-owned-by-other-users_{context}']
+= Accessing notebook servers owned by other users
+
+[role='_abstract']
+Administrators can access notebook servers that are owned by other users to correct configuration errors or help a data scientist troubleshoot problems with their environment.
+
+.Prerequisites
+* You are part of the {openshift-platform} administrator group. See _Adding administrative users for {openshift-platform}_ for more information.
```

Comment:
I don't think this non-link will work for the upstream. Should it be the same as https://github.com/opendatahub-io/opendatahub-documentation/pull/32/files#diff-0940563d4d2b6319e82aa57d3f8c5171913381be415babfebda69742d80fe272R10?

Reply:
fixed